msteams-platform/get-started/first-message-extension.md

```diff
@@ -162,6 +162,23 @@ Deployment involves provisioning resources on an active Azure subscription and d
 
 </details>
 
+## Add a configuration page to your messaging extension on GitHub
+
+[!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
+
+## Code sample
+
+The Teams Search Auth Config for sample projects on GitHub, demonstrate how to create messaging extensions that include a configuration page and Bot Service authentication. The samples also demonstrate how to create message extensions that accept search requests and return the results after the user has signed in.
+
+| **Sample name** | **Description** | **.NET** | **Node.js** | **Python** |
+|-----------------|-----------------|-------------|--------------|--------|
+| Bot builder | To create messaging extensions. | [View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/52.teams-messaging-extensions-search-auth-config) | [View](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/52.teams-messaging-extensions-search-auth-config) | [View]( https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/python/50.teams-messaging-extension-search) |
+
+## Additional code sample
+
+> [!div class="nextstepaction"]
+> [View more Bot Framework Samples on GitHub](https://github.com/microsoft/BotBuilder-Samples)
```

Comment:
Could you please link to this section? https://github.com/OfficeDev/microsoft-teams-samples#messaging-extensions-samples-using-the-v4-sdk